herddb-jdbc/src/test/java/herddb/jdbc/SystemTablesTest.java

```diff
@@ -266,7 +266,7 @@ public void ensureWeThrowAppropriateExceptionWhenNotNullConstraintViolated_AddBa
                     insertPs.executeBatch();
                 } catch(SQLException ex) {
                     assertTrue(ex.getMessage().contains("StatementExecutionException"));
-                    assertTrue(ex.getMessage().contains("Cannot have null value in non null type string not null"));
```

Comment:
we should reword the exception message: `cannot have a null for columns of type 'string not null'`